providers/tests/system/amazon/aws/example_mwaa.py

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow.models.baseoperator import chain
+from airflow.models.dag import DAG
+from airflow.providers.amazon.aws.operators.mwaa import MwaaTriggerDagRunOperator
+
+from providers.tests.system.amazon.aws.utils import SystemTestContextBuilder
+
+DAG_ID = "example_mwaa"
+
+# Externally fetched variables:
+EXISTING_ENVIRONMENT_NAME_KEY = "ENVIRONMENT_NAME"
+EXISTING_DAG_ID_KEY = "DAG_ID"
+
+
+sys_test_context_task = (
+    SystemTestContextBuilder()
+    # NOTE: Creating a functional MWAA environment is time-consuming and requires
+    # manually creating and configuring an S3 bucket for DAG storage and a VPC with
+    # private subnets which is out of scope for this demo. To simplify this demo and
+    # make it run in a reasonable time, an existing MWAA environment with is required
```

Comment:
I think you got squirreled here.  "an existing MWAA environment with is required".   You forgot something there.